### DISCUSSION TITLE
I want to get the suggestion providing the needs and wanted to compute the field undefined

### DISCUSSION THREAD

**Discussion by broisnischal** [prisma/prisma#23150] [Q&A]
### Question

```ts
import { Prisma } from "@prisma/client";

export default Prisma.defineExtension({
	name: "Undefine Field",
	result: {
		$allModels: {
			undefineField: {
				needs: {},
				compute() {
					return undefined;
				},
			},
		},
	},
});

```

**Comment by nurul3101** [comment]
Hi @broisnischal 👋 

Can you please elaborate on your use case? What are you trying to achieve with this extension? I didn't understand your use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
